### PR TITLE
Missing translation

### PR DESCRIPTION
**Pull request by tamplan**
Hi,

I change the html tag lang attribute to be the locale selected in the application config file.

I update the french translation file too.

Next pull request i plan is to translate some hardcoded strings in views files.

Regards,

### CODE REVIEW

**Comment by arcanedev-maroc** [comment]
Hi @tamplan,

Thanks for your work, but can you split this PR into multiple PRs with single purpose.

As you can see, the title of this PR is `Missing translation` but you're also editing CSS and views.

Merci 👍

**Comment by tamplan** [comment]
Ok, i'll try to do this, try because I do not feel still comfortable in use of git for the moment.

**Comment by tamplan** [comment]
I have reverted the commit and made a new branche.

Regards

**Comment by arcanedev-maroc** [comment]
OK, now it make more sense.

I'm going to review all your changes and fix things when i got some spare time. 

This is a HUGE PR and i've moved it to `translate-all` branch instead of `master` for BC.

Great job and thanks again for your suggestion 👍

File: resources/lang/en/general.php [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+    'logs'            => 'Logs',
+    'log'             => 'log',
+    'log_info'        => 'log informations',
+    'download'        => 'Download',
+    'delete'          => 'Delete',
+    'file_path'       => 'File Path :',
+    'log_entries'     => 'Log Entries :',
+    'size'            => 'Size :',
+    'created_at'      => 'Created At :',
+    'updated_at'      => 'Updated At :',,
```
Comment (arcanedev-maroc) [inline resources/lang/en/general.php]: Can you remove the duplicated `,` ?

Check: https://travis-ci.org/ARCANEDEV/LogViewer/jobs/260658666#L394

Comment (tamplan) [inline resources/lang/en/general.php]: No, sorry, I make a translate-all branch  too in my repo and update the file...

**Pull request merged**